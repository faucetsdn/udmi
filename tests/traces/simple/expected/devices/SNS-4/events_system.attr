{
  "deviceNumId" : "00000079057151",
  "publishTime" : "1999-10-20T01:02:03Z",
  "subFolder" : "system",
  "deviceRegistryId" : "ZZ-TRI-FECTA",
  "subType" : "events",
  "msgSource" : "003_events_system.json",
  "deviceId" : "SNS-4",
  "projectId" : "playback-project"
}