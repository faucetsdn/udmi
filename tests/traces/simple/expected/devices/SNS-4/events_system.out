{
  "timestamp" : "1999-10-20T01:02:03Z",
  "version" : "1.5.2",
  "sub_folder" : "system",
  "sub_type" : "events",
  "status" : {
    "message" : "While converting to json node: 1 schema violations found",
    "detail" : "events_system: While converting to json node: 1 schema violations found @Validator.validateMessage(REDACTED); 1 schema violations found; /logentries/0/category: instance entry category not recognized",
    "category" : "validation.device.schema",
    "timestamp" : "1999-10-20T01:02:03Z",
    "level" : 500
  },
  "errors" : [ {
    "message" : "While converting to json node: 1 schema violations found",
    "detail" : "events_system: While converting to json node: 1 schema violations found @Validator.validateMessage(REDACTED); 1 schema violations found; /logentries/0/category: instance entry category not recognized",
    "category" : "validation.device.schema",
    "timestamp" : "1999-10-20T01:02:03Z",
    "level" : 500
  } ]
}
